src/System.Net.Requests/ref/System.Net.Requests.cs

```diff
@@ -291,17 +331,17 @@ public enum WebExceptionStatus
         Success = 0,
         Timeout = 14,
         TrustFailure = 9,
-        UnknownError = 16
+        UnknownError = 16,
```

Comment:
nit: why does the tooling add a comma to the last element like this?

Reply:
> why does the tooling add a comma to the last element like this?

So that when new enum members are added in the future the diff only shows new values, not also "and I had to add a comma here".